site/content/blog/2022-10-01-whats-new-in-svelte-october-2022.md

```diff
@@ -0,0 +1,136 @@
+- New types for `ComponentEvents` and `SveltePreprocessor` (**3.50.0**)
+- Improved parsing speed when encountering large blocks of whitespace (**3.50.0**)
+- All global JavaScript objects and functions are now recognized as known globals (**3.50.1**)
+
+For all the changes to the Svelte compiler, including upcoming changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## New in Language Tools
+- Better handling of configs for code formatting (**106.0.0**, [#1598](https://github.com/sveltejs/language-tools/pull/1598))
+- Easily create sveltekit route files from the context menu or command palette (**106.1.0**, [#1620](https://github.com/sveltejs/language-tools/pull/1620))
+
+---
+
+## Ask Questions in the Svelte Discord
+
+In case you missed the announcement, the Svelte Discord has an exciting new update... a forum! The new [questions channel](https://discord.com/channels/457912077277855764/1023340103071965194) utilizes Discord's new forums feature to help the community better ask, find and answer questions!
+
+It can be used for anything you may be trying to accomplish using Svelte including using SvelteKit, community libraries, tools, etc. So ask away!
+
+---
+
+## Community Showcase
+
+**Apps & Sites built with Svelte**
+- [Timeflow](https://www.timeflow.site/) is a smart calendar & task manager that dynamically schedules your tasks between your events
+- [GeoQuest](https://github.com/woutdp/geoquest) is an open source geography game
+- [Houses Of](https://housesof.world/) is a project showcasing charismatic houses around the world
+- [Greenfield Brewery](https://greenfield-brewery.vercel.app/) is a tool for quickly installing a lot of homebrew casks
+- [Gram Jam](https://gramjam.app/) is a word puzzle game inspired by match three games and Scrabble
+- [Beatbump](https://github.com/snuffyDev/Beatbump) is a privacy-respecting alternative frontend for YouTube Music
+- [RoomOS Device Widgets](https://github.com/wxsd-sales/roomos-device-widgets) is an app for demoing RoomOS device capabilities in Kiosk/PWA mode
+- [World Seed](https://store.steampowered.com/app/1870320/World_Seed/) is a full blown online multiplayer game
+- [Lirify](https://lirify-tan.vercel.app/) is a song lyrics writing web app tool made in Latvia
+- [Splet Tech Konferencija](https://www.splet.rs/) is a tech conference in Serbia with a *very* fancy website
+- [Unbounded](https://unbounded.polkadot.network/) is an open-source variable font - funded by blockchain (and an awesome-looking website)
+- [Porter's Paints](https://shop.porterspaints.com/) is an eCommerce site for (you guessed it) paints built with Svelte
+- [CRAN/E](https://www.cran-e.com/) is a search engine for modern R-packages 
+
+
+**Learning Resources**
+
+_Starring the Svelte team_
+- [Upgrading SvelteKit](https://www.youtube.com/watch?v=vzeZskhjoeQ) by Svelte Sirens (featuring Kev and GHOST!)
```

Comment:
`- [Upgrading SvelteKit](https://www.youtube.com/watch?v=vzeZskhjoeQ) by Svelte Sirens (with Brittney, Kev, and GHOST!)`